### ISSUE TITLE
Scaling issue with tool window tabs on HiDPI screens

### ISSUE THREAD

**Issue by vemilyus** [mallowigi/material-theme-jetbrains-eap#82]
## Problem
The height of the tabs on tool windows is insufficient and cuts of the labels. Please see the attached screenshot.

![hidpipaneltitles](https://user-images.githubusercontent.com/7115009/28541067-e62a8846-70b7-11e7-9ac8-9647d9ac4233.png)

## Material Theme Version
```
Material Theme UI EAP 0.6.0.1
```

## IntelliJ Version
```
IntelliJ IDEA 2017.2.1 EAP
Build #IU-172.3544.6, built on July 19, 2017
JRE: 1.8.0_152-release-915-b6 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.10.0-28-generic
```

## OS
```
Memory: 31,3 GB
Processor: Intel® Core™ i7-5960X CPU @ 3.00GHz × 16 
Graphics: GeForce GTX TITAN X/PCIe/SSE2
Base system: Ubuntu 17.04 64-bit
```

**Comment by mallowigi** [comment]
fixing

**Comment by vemilyus** [comment]
Confirmed fixed in 0.7.0 :D You rock! :tada:
